python_modules/libraries/dagster-dbt/dagster_dbt/__init__.py

```diff
@@ -174,19 +153,59 @@
                 "load_assets_from_dbt_project",
                 "dagster_dbt.asset_defs",
                 (
-                    "\n\nTo generate a dbt manifest for @dbt_assets at run time using `dbt parse`,"
+                    ".\n\nTo generate a dbt manifest for @dbt_assets at run time using `dbt parse`,"
                     " see the reference:"
-                    " https://docs.dagster.io/integrations/dbt/reference#loading-dbt-models-from-a-dbt-project.",
+                    " https://docs.dagster.io/integrations/dbt/reference#loading-dbt-models-from-a-dbt-project"
                 ),
             ),
         ]
     },
 }
 
+_DEPRECATED_WARNING: Final[Mapping[str, Tuple[str, str, str]]] = {
+    ##### EXAMPLE
+    # "Foo": (
+    #     "dagster.some.module",
+    #     "1.1.0",  # breaking version
+    #     "Use Bar instead.",
+    # ),
+    **{
+        value: (
+            module,
+            "0.24.0",
+            (
+                "Use the @op decorator and DbtCliResource instead.\n\n"
```

Comment:
Does this warning also display in logs? If not, could we clean up some of the formatting so it's more readable in the docs? It's a pretty big blob at the moment:

![Screenshot 2024-01-10 at 12 26 21 PM](https://github.com/dagster-io/dagster/assets/16615212/980d31b5-c99a-4fd4-9b4c-fe5b324562b9)

Reply:
https://github.com/dagster-io/dagster/pull/19151 should resolve